components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -53,6 +55,13 @@ export class Panel extends React.Component<Props, State> {
 
     this.actions.getWalletProperties()
     this.actions.getCurrentReport()
+
+    let checkValue = new Promise(resolve => chrome.braveRewards.getNonVerifiedSettings(resolve))
```

Comment:
Just curious, why are we using this pattern? Is there a reason that we can't just have an `ON_VERIFIED_SETTINGS` action and just update it then?

Reply:
we use direct response as this is a simple getter and there is no async stuff happening